packages/grid_client/src/primitives/nodes.ts

```diff
@@ -409,7 +409,7 @@ class Nodes {
   async nodeAvailableForTwinId(nodeId: number, twinId: number): Promise<boolean> {
     return send("get", urlJoin(this.proxyURL, `/nodes/${nodeId}`), "", {})
       .then(node => {
```

Comment:
can we add the new field in the `NodeInfo` interface?

```suggestion
      .then(node: NodeInfo => {
```